software/apps/cholesky_schedule_q32s/initialization.h

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 ETH Zurich and University of Bologna.
```

Comment:
This file is identical to the `initialization.h` file in the other app, right? Is there a reason to keep two copies, or could we move it to the runtime and have both apps include the same one?

Reply:
Yes, it is the same, we can move it to "kernels", but I am starting to generate the inputs using Python scripts in the apps folder. Ideally, each app should have something to generate its own inputs and outputs, having twice the same initialization.h file here is just a coincidence here.